_build/scripts/setup.py

```diff
@@ -62,6 +66,13 @@ def run(shouldClean):
   # libraries like Windows SDK, C++ standard library is always required for development, where as libraries like ImGui
   # are not required and can always be replaced if a better library comes along.
   regis.install_externals.run()
+
+  # This setup is optional, but for users using VSCode as an IDE we copy cover a "tasks" files
```

Comment:
typo in comment "cover" -> "over"
also weird sentence "a tasks files", it's either "a tasks file" or "tasks files"